backend/multi/backend.c

```diff
@@ -140,15 +140,6 @@ bool wlr_multi_backend_add(struct wlr_backend *_multi,
 		return true;
 	}
 
-	struct wlr_renderer *multi_renderer =
```

Comment:
I'd really like to keep this.

Reply:
If I return here (no matter if I return true or false) then the further code for allocating subbackend is not called and this is leading to only support one GPU...

Reply:
You can add `backend_renderer != multi_renderer`